src/FsCodec.SystemTextJson/TypeSafeEnumConverter.fs

```diff
@@ -1,5 +1,6 @@
-﻿namespace FsCodec.SystemTextJson
+﻿namespace FsCodec.SystemTextJson.Converters
```

Comment:
not sure about this as it clashes with two other conventions:
- having namespaces be justified by having sufficient content (while STJ has a `.Converters` subnamespace, it also has a lot of Converters (when contrasted with STJ itself))
- 1:1 mapping between newtonsoft and STJ impls (where the Converters are not in a subnamespace)

slightly tangential, but this is very good and relevant rules from here are definitely borned in mind https://azure.github.io/azure-sdk/dotnet_introduction.html

Reply:
[`JsonOptionConverter`](https://github.com/NickDarvey/FsCodec/blob/aab8dce0240469351172c43e962aaaa8b57f111f/src/FsCodec.SystemTextJson/JsonOptionConverter.fs#L1) and [`JsonRecordConverter`](https://github.com/NickDarvey/FsCodec/blob/aab8dce0240469351172c43e962aaaa8b57f111f/src/FsCodec.SystemTextJson/JsonRecordConverter.fs#L1) were in the `FsCodec.SystemTextJson.Converters` namespace so I updated `TypeSafeEnumConverter` and `UnionConverter` to match.

I will go ahead and make the namespace `FsCodec.SystemTextJson` for all four of those converters.

Reply:
Ah, sorry for the confusion - Yaron did the impl of those (originally in the Equinox repo), I just moved them (it's not entirely unreasonable to nest them like that of course)

The other thing that makes it subjective is that neither converter should come into play in normal usage (they get added in Options.Create; you don't normally touch them directly)